torchtext/__init__.py

```diff
@@ -1,11 +1,13 @@
 import os
+from ._extension import _init_extension
+
+_init_extension()
```

Comment:
Please check the compatibility of this initialization update on fbcode.

Also I recommend to call this function in `_extension.py` just for the sake of the simplification of `__init__.py`. (delegating the reponsibility to `_extension.py`, which yields a better locality.)

Still, the `_extension` module has to be imported first in `__init__.py`.

cc @parmeet

Reply:
Updated the function call to reside within `_extension.py`. Will import this entire PR and test on fbcode before landing once we get all GH CI tests passing